.github/workflows/php-test-plugins.yml

```diff
@@ -16,6 +16,7 @@ on:
       - 'tests/multisite.xml'
       - 'composer.json'
       - 'composer.lock'
+      - 'webpack.config.js'
```

Comment:
The reason for adding this is that unit tests for the plugins can fail if the minified assets aren’t generated. Since Webpack is responsible for generating these assets and copying necessary libraries from `node_modules` to the build, it makes sense to include changes to `webpack.config.js` as a trigger for the unit tests.